### ISSUE TITLE
extra spacing under news section on tablet view (screen size < 978 )

### ISSUE THREAD

**Issue by Bhavika-Gianey** [ocaml/v2.ocaml.org#1387]
There is extra spacing under the news section of the Home page on  tablet view i.e. screen size less than 978 px 

## On large view 
![old3-2](https://user-images.githubusercontent.com/63343906/113474440-9e2d4a80-948d-11eb-9959-9ba59dd32533.PNG)

## On medium view
![old3-1](https://user-images.githubusercontent.com/63343906/113474453-aa190c80-948d-11eb-819f-c48a7479b63c.PNG)

I would like to fix the spacing problem here.

**Comment by pitag-ha** [comment]
What do you have in mind to improve this? :)

**Comment by Bhavika-Gianey** [comment]
We can increase the padding of each list item in the news section so that it covers the unused space.

Eg:

![ocaml org_ (2)](https://user-images.githubusercontent.com/63343906/114055581-85b69900-98ae-11eb-9653-9d5276d3980d.png)

**Comment by pitag-ha** [comment]
Yes, that would work :) There is a [PR](https://github.com/ocaml/ocaml.org/pull/1420) open at the moment changing that layout though. So we should wait to avoid stepping on each other's toes.

And thanks for your contributions so far, @Bhavika-Gianey , they've been great! Now, it's time for you to move forward to a harder issue. See [here](https://github.com/ocaml/ocaml.org/issues/1245#issue-821019972) and [here](https://github.com/ocaml/ocaml.org/issues/1467#issuecomment-815117089). I think this issue can be considered simple.

**Comment by Bhavika-Gianey** [comment]
> Yes, that would work :) There is a [PR](https://github.com/ocaml/ocaml.org/pull/1420) open at the moment changing that layout though. So we should wait to avoid stepping on each other's toes.
> 
Sure @pitag-ha, actually I opened the issue earlier than the specified one so was I was confused whether to carry this issue forward or not. Shall I close it now??

**Comment by Bhavika-Gianey** [comment]
> And thanks for your contributions so far, @Bhavika-Gianey, they've been great! Now, it's time for you to move forward to a harder issue. See [here](https://github.com/ocaml/ocaml.org/issues/1245#issue-821019972) and [here](https://github.com/ocaml/ocaml.org/issues/1467#issuecomment-815117089). I think this issue can be considered simple.

Thanks, @pitag-ha :). I would love to learn and work on harder issues. I have seen all the medium issues are already assigned so if you come across any new issues, kindly let me know :).

**Comment by pitag-ha** [comment]
>  actually I opened the issue earlier than the specified one so was I was confused whether to carry this issue forward or not.

yes, you're totally right! I'm also not sure if @guptadiksha307's PR will be merged. That's a general risk and even higher when opening PRs before discussing the idea as done in that PR.

If you want, you can also open a PR for your idea. In projects that lots of people are working on, it sometimes happens that two PR's aren't compatible and max one of them will be merged. So I'd say it's up to you :)
